### PR TITLE
MSAL-Angular Angular13 Support & RxJS 6/7 Sample Apps

### PR DESCRIPTION
**Pull request by Robbie-Microsoft**

### CODE REVIEW

**Review by jasonnutter — CHANGES_REQUESTED** [review]
Tested locally and this looks good, thanks! I've push up one commit to remove the `package-lock` files. 

Please complete the following:

1. Generate change file (can mark as minor change)
2. Add both of these samples to the e2e tests (@jo-arroyo can help you get that running)
3. Add `13` as supported verison in the supported version table in the MSAL Angular README
4. As fast follow up to this PR, start another PR to add an Angular 14 sample.

**Review by jo-arroyo — CHANGES_REQUESTED** [review]
Also tested locally. Great work! A few more changes would be nice:

1. Update the READMEs for each sample, cf. Angular 10 sample. 
2. Update the README for the `msal-angular-v2-samples` folder. This is where we list the differences between the samples. Feel free to include an explanation for why we have a RxJS 6 and RxJS 7 sample.
3. Update the MSAL Angular README to include links to angular13 samples.
4. Remove the `.vscode` and `assets` folders for each sample. 

I will reach out to you offline about the e2e tests.

**Comment by Robbie-Microsoft** [comment]
>

File: change/@azure-msal-angular-8f602981-ceaf-457b-9710-7afdfeb9fcae.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Created two Angular13 sample apps. One uses RxJS v6 and the other uses RxJS v7.",
```
Comment (jasonnutter) [inline change/@azure-msal-angular-8f602981-ceaf-457b-9710-7afdfeb9fcae.json]:
```suggestion
  "comment": "Add support for rxjs v7 and Angular 13 #4605",
```

**Comment by jasonnutter** [comment]
@Robbie-Microsoft E2E tests are passing. It required that I add `compilerOptions.paths` entries for Angular to both samples (which do in the other samples), and for rxjs for the rxjs7 sample. Example:

https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/94f17e2d9529343a2d0c543dbd2376b5f2ec75c9/samples/msal-angular-v2-samples/angular13-rxjs7-sample-app/tsconfig.json#L25-L28

It would probably be good to call this out for each sample.

**Review by jasonnutter — APPROVED** [review]
Looks good, thanks for putting this together!

File: samples/msal-angular-v2-samples/angular13-rxjs7-sample-app/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Angular13RxJS7SampleApp
```
Comment (jo-arroyo) [inline samples/msal-angular-v2-samples/angular13-rxjs7-sample-app/README.md]: Can you update these samples to have a similar README with as other msal-angular samples? See here: https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v2-samples/angular10-sample-app

File: samples/msal-angular-v2-samples/angular13-rxjs6-sample-app/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Angular13RxJS6SampleApp
```
Comment (jo-arroyo) [inline samples/msal-angular-v2-samples/angular13-rxjs6-sample-app/README.md]: Requesting readme change here to align with other msal-angular sample readmes: https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-angular-v2-samples/angular10-sample-app

File: samples/msal-angular-v2-samples/angular13-rxjs7-sample-app/jest.config.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    displayName: "angular13-rxjs7-sample-app",
+    globals: {
+        __PORT__: 4208,
```
Comment (jo-arroyo) [inline samples/msal-angular-v2-samples/angular13-rxjs7-sample-app/jest.config.js]: Change to unique ports (different to other msal-angular samples).

File: samples/msal-angular-v2-samples/angular13-rxjs6-sample-app/jest.config.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    displayName: "angular13-rxjs6-sample-app",
+    globals: {
+        __PORT__: 4208,
```
Comment (jo-arroyo) [inline samples/msal-angular-v2-samples/angular13-rxjs6-sample-app/jest.config.js]: Change to unique ports (different to other msal-angular samples).

**Review by jo-arroyo — CHANGES_REQUESTED** [review]
A few changes, thanks!

**Review by jo-arroyo — APPROVED** [review]
Looks great! Thanks for all your good work!

**Pull request merged**